src/index.ts

```diff
@@ -157,6 +164,7 @@ function wrapTriggerAction<V, P, T extends string, TA extends PayloadActionCreat
 
   const updatedTrigger = <TriggerActionCreator<V, P, T, SagaPromiseMeta<V, T>, TA>>createAction(type, (...args: any[]) => merge(triggerAction.apply(null, args), {
     meta: <SagaPromiseMeta<V, T>>{
+      actionSymbol: promiseActionSymbol,
```

Comment:
One idea of symbols is to use a symbol as property key to have a unique property key that only we, the authors of the package can associate to and for example check for uniqueness of an object. Instead of "actionSymbol" = promiseActionSymbol, you would rather use [promiseActionSymbol] = true. 🙂

Reply:
@teneko fixed